mod_root/BepInEx/plugins/BetterTrader/item_defaults.cfg

```diff
@@ -26,20 +26,42 @@ If tradeable = false, then even after you've discovered the item it still won't
 IF THE WaitForDiscovery OPTION ISN'T SHOWING UP, BE SURE TO RUN THE GAME AT LEAST ONCE WITH BetterTrader INSTALLED! THE OPTION WILL BE GENERATED ON STARTUP!
 
 #Better Trader#
+<<<<<<< HEAD
```

Comment:
looks like a merge error.  Not sure how this happened exactly.  I will remediate.